### PR TITLE
Update implicit animations page to use current code examples

### LINKED ISSUE #6897: [PAGE ISSUE]: 'Implicit animations' contains analyzer errors
### Page URL

https://docs.flutter.dev/codelabs/implicit-animations/

### Page source

https://github.com/flutter/website/tree/main/src/codelabs/implicit-animations.md

### Describe the problem

The codelab iframes for this codelab contain analyzer warnings. Most are simple lints, by the looks of it.

### Expected fix

We should update the code to be warning-free.

### Additional context

We should also figure out how to get this code under test.

### PR DESCRIPTION
**Pull request by micpap25**
Fixes the implicit animation page's linting error by updating the iframe code to use the correct starter / complete code from the examples folder. 
Fixes #6897

### CODE REVIEW

File: src/_includes/docs/implicit-animations/shape-shifting-starter-code.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ```run-dartpad:theme-light:mode-flutter:split-60:width-100%:height-500px:ga_id-shape_shifting_starter_code
 {$ begin main.dart $}
+// Copyright 2019 the Dart project authors. All rights reserved.
```
Comment (domesticmouse) [inline src/_includes/docs/implicit-animations/shape-shifting-starter-code.md:3]: We probably don't need the copyright headers in the published code

File: src/_includes/docs/implicit-animations/shape-shifting-complete.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 ```run-dartpad:theme-light:mode-flutter:split-60:width-100%:height-500px:ga_id-shape_shifting_complete
 {$ begin main.dart $}
+// Copyright 2019 the Dart project authors. All rights reserved.
```
Comment (domesticmouse) [inline src/_includes/docs/implicit-animations/shape-shifting-complete.md:3]: ditto

File: src/_includes/docs/implicit-animations/fade-in-starter-code.md [hunk 3]
```diff
@@ -1,43 +1,49 @@
 ```run-dartpad:theme-light:mode-flutter:split-60:width-100%:height-532px:ga_id-fade_in_starter_code
 {$ begin main.dart $}
+// Copyright 2019 the Dart project authors. All rights reserved.
```
Comment (domesticmouse) [inline src/_includes/docs/implicit-animations/fade-in-starter-code.md:3]: ditto

File: src/_includes/docs/implicit-animations/fade-in-complete.md [hunk 4]
```diff
@@ -1,34 +1,42 @@
 ```run-dartpad:theme-light:mode-flutter:split-60:width-100%:height-532px:ga_id-fade_in_complete
 {$ begin main.dart $}
+// Copyright 2019 the Dart project authors. All rights reserved.
```
Comment (domesticmouse) [inline src/_includes/docs/implicit-animations/fade-in-complete.md:3]: ditto

**Review by domesticmouse — CHANGES_REQUESTED** [review]
Please drop the copyright headers. Otherwise, LGTM

**Comment by micpap25** [comment]
Hi, I'll drop them when I get the chance. Should I then also drop them from the diffs on the rest of the page?

**Comment by domesticmouse** [comment]
> Hi, I'll drop them when I get the chance. Should I then also drop them from the diffs on the rest of the page?

Yes please!

**Comment by sfshaza2** [comment]
Thanks for updating this, @micpap25!

**Comment by micpap25** [comment]
I made the updates; I can't tell what's causing the copyright headers to show up in the diffs. The comment below the copyright header is necessary though.

**Comment by domesticmouse** [comment]
@khanhnwin can you help plz?

**Comment by khanhnwin** [comment]
hey y'all - I don't seem to see the Copyright text in the diffs anymore. Am I missing something?

**Comment by parlough** [comment]
@domesticmouse's review comments on those lines will still show the headers from previous commits, but they are no longer there in the actual diff. I've gone ahead and resolved the outdated comments.

Looks good to me!

**Review by domesticmouse — APPROVED** [review]

**Pull request merged**